File: src/main/java/com/example/ssauc/user/history/dto/SellHistoryCompletedDto.java
package com.example.ssauc.user.history.dto;

import java.time.LocalDateTime;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class SellHistoryCompletedDto {
    private Long orderId;
    private Long productId;
    private String productName;
    private String productImageUrl;
    private String buyerName;
    private String profileImageUrl;
    private Long totalPrice;
    private LocalDateTime orderDate;
    private LocalDateTime completedDate;
}

File: src/main/java/com/example/ssauc/user/history/dto/SellHistoryCompletedDto.kt
package com.example.ssauc.user.history.dto

import lombok.AllArgsConstructor
import lombok.Builder
import lombok.Data
import lombok.NoArgsConstructor
import java.time.LocalDateTime

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
class SellHistoryCompletedDto {
    public val orderId: Long? = null
    public val productId: Long? = null
    public val productName: String? = null
    public val productImageUrl: String? = null
    public val buyerName: String? = null
    public val profileImageUrl: String? = null
    public val totalPrice: Long? = null
    public val orderDate: LocalDateTime? = null
    public val completedDate: LocalDateTime? = null
}
